examples/slack-connector/src/main/resources/application.properties

```diff
@@ -0,0 +1,47 @@
+########################################################################
+#Properties used by channel connector connector framework.
+########################################################################
+
+#The base URL for Chatbot Runtime.
+sfdc.einstein.bots.runtime-url=${BOT_RUNTIME_URL:https://runtime-api-na-west.stg.chatbots.sfdc.sh}
+
+#The name of the integration that will be used. This should match the Integration Name used
+#when adding connected app API connection to Bot.
+sfdc.einstein.bots.integration-name=${BOT_INTEGRATION_NAME:sfdc.chatbot.api}
+
+#OAuth Properties
+#Path to the private key needed to get OAuth Tokens.
+sfdc.einstein.bots.oauth.private-key-file=${OAUTH_PRIVATE_KEY_FILE_PATH:/app/target/classes/private_key.der}
+
+#The URL to hit to obtain an OAuth Token
+sfdc.einstein.bots.oauth.login-endpoint=${LOGIN_END_POINT:https://login.test1.pc-rnd.salesforce.com/}
```

Comment:
change this to login.salesforce.com. Refer to twitter connector for reference

Reply:
Changed to login.salesforce.com